README.md

```diff
@@ -146,3 +146,88 @@ cd end-to-end-tests
 yarn install
 yarn run test-webdriver-manager
 ```
+
+## Local cBioPortal database for e2e-tests
+To enable e2e-tests on for features that depend on data that are not included in studies served by the public cBioPortal instance, cbioportal-frontend provides the `e2e local database` (refered to as _e2e-localdb_ below) facility that allows developers to load custom studies used for e2e-tests. CircleCI runs the `e2e local database` tests as a separate job.
```

Comment:
i'm curious how long does it take to create db and start server?

Reply:
For e2e tests this takes ~12mins. total.
For e2e-localdb tests it takes ~17mins total.

I guess that we may also improve on performance to create changes of the database.